src/transform3d.rs

```diff
@@ -177,6 +177,18 @@ where T: Copy + Clone +
         )
     }
 
+    /// Check whether shapes on the XY plane with Z pointing towards the
+    /// screen transformed by this matrix would be facing back.
+    pub fn is_backface_visible(&self) -> bool {
+        // inverse().m33 < 0;
```

Comment:
Not terribly useful to understand the underlying math but this is ported from gecko's code apparently: https://searchfox.org/mozilla-central/rev/48ea452803907f2575d81021e8678634e8067fc2/gfx/2d/Matrix.h#1392